src/assets/projectWeekBannerAssets.tsx

```diff
@@ -26,20 +20,15 @@ export const projectWeekPupilText = (
     </a>{' '}
     findest.
     <br />
-    Stöbere hier durch das Angebot der Projektwoche und melde dich kostenlos zu
+    Stöbere hier durch das Angebot der anstehende Fokuswoche und melde dich kostenlos zu
     den digitalen Angeboten an!
   </>
 );
 
 export const projectWeekStudentText = (
   <>
-    Vom 21.-25. März gibt es ein buntes Programm rund um das Thema{' '}
-    <b>Abschluss und dann?</b>. Schüler:innen können sich dort mit verschiedenen
-    Themen rund um Studium, Ausbildung und Freiwilligendienst, aber auch
-    Bewerbungen und Studienfinanzierung auseinandersetzen.
-    <br />
-    Dies findet im Rahmen des neuen Programms &quot;Lern-Fair Fokus &ndash; die
-    digitalen Projektwochen&quot; statt, wozu du mehr Informationen auf unserer
+    Hier finden Kurse im Rahmen des neuen Programms &quot;Lern-Fair Fokus &ndash; die
```

Comment:
@Maascha I also adapted the text for students to "Fokuswoche", and removed the specific content (as I don't think we need to change that every month?) ... Objections?

Reply:
it would be good, if there is focus week specific content (say, the name of the week), which we update every month. But you changed that at least somewhere. Where is the specific content removed?

Reply:
yes, it is okay, Although I would prefer a text where we can see each date, so the students know when to take action

**Lern-Fair Fokus**
In den Fokuswochen von [Lern-Fair Fokus](www.lern-fair.de/helfer/fokus) gibt es ein buntes Programm für interessierte Schüler:innen rund um ein bestimmtes Themengebiet. Für 2022 sind folgende Themenschwerpunkte geplant:

- **Abschluss und dann?** 21.03. - 25.03.
- **Lernen lernen** 09.05. - 13.05. 
-  **Kommunikation** 27.06. - 01.07.
- **Klima und Nachhaltigkeit** 12.09. - 16.09.
- **Abschluss und dann?** 03.10. - 07.10.
- **MINT-Projekte & KI** 07.11. - 11.11.

Erstelle hier einen Kurs für eine der Fokuswochen und stöbere durch das Angebot der anstehenden Fokuswoche. Außerdem kannst du deinen eigenen eingestellten Kurs finden, sobald dieser freigeschaltet wurde.
Schreibe uns deine Ideen, deine Frage oder deine Anmerkungen per Mail an [fokus@lern-fair.de](mailto:fokus@lern-fair.de).

Reply:
Like [this](https://cs-web-user-app-pr-232.herokuapp.com/login?token=authtokenS1) ?